pkgs/development/libraries/at-spi2-core/default.nix

```diff
@@ -62,6 +62,11 @@ stdenv.mkDerivation rec {
     glib
   ];
 
+  patches = [
```

Comment:
GNOME packages prefer having `patches` just after `src`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.